core/domain/subtopic_page_domain.py

```diff
@@ -137,23 +160,48 @@ def from_dict(cls, page_contents_dict):
         Returns:
             SubtopicPageContents. The corresponding object.
         """
-        page_contents = state_domain.SubtitledHtml.from_dict(
+        page_contents = state_domain.SubtitledHtml.from_dict( # type: ignore[no-untyped-call]
             page_contents_dict['subtitled_html'])
         page_contents.validate()
         return cls(
             page_contents,
-            state_domain.RecordedVoiceovers.from_dict(page_contents_dict[
+            state_domain.RecordedVoiceovers.from_dict( # type: ignore[no-untyped-call]
+            page_contents_dict[
```

Comment:
```suggestion
            page_contents_dict['recorded_voiceovers']),
```

and remove the line below, ditto below